daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/infoset/JDOMInfosetOutputter.scala

```diff
@@ -46,7 +49,8 @@ class JDOMInfosetOutputter extends InfosetOutputter with XMLInfosetOutputter {
     result = Maybe(root.asInstanceOf[org.jdom2.Document])
   }
 
-  def startSimple(diSimple: DISimple): Unit = {
+  override def startSimple(se: InfosetSimpleElement): Unit = {
+    val diSimple = se.asInstanceOf[DISimple]
```

Comment:
Is casting to a DISimple done to avoid having to update other outputters and deal with that later? I assume the goal is these infoset outputters should only use InfosetFooEelements? Doing the conversion would be good evidence that the InfosetElement API interface makes everything needed available.